### PR TITLE
Add a simple e2e test

### PR DESCRIPTION
**Pull request by kirjs**
I'm using [nightwatch.js](http://nightwatchjs.org/) which is a framework agnostic selenium wrapper. 

If this looks good, I'm planning to: 
1. Add more tests
2. Update the docs 
3. Add Github integration to run on every commit. 

Here is a quick demo:
aug-13-2016 21-04-53

### CODE REVIEW

**Comment by remy** [comment]
Works for me. Maybe on a prepush rather than commit since launching takes time, but makes sense.

The big question though: will this work inside of a Travis CI environment?

**Comment by kirjs** [comment]
Yes it can work with Travis CI, here are [some docs](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-Sauce-Labs). 

Their recommendation is to use SauceLabs, which is [free](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-Sauce-Labs) for open-source projects.

**Comment by remy** [comment]
I'm just making a note (for myself) that I should list some of the core tests that I've been performing manually to check that things aren't broken. I think that would make a great baseline for tests.

The first, and most important though: when the code is changed in an empty bin, is it saved? You can tell by there being a change in the URL.

File: docs/running-your-own-jsbin.md [hunk 1]
```diff
@@ -103,3 +103,11 @@ Finally, ensure either the config.local.json's `env` property is set to "product
     $ NODE_ENV=production node .
 
 And that's it.
+
+
+## End-to-end tests
+JSBin has end-to-end tests. The tests can be found in `tests/e2e` folder.
```
Comment (remy) [inline docs/running-your-own-jsbin.md]: nitpick: should be `JS Bin` or `jsbin`.

**Comment by kirjs** [comment]
@remy I'll need access to the travis-CI panel, or your help with setting it up.

**Comment by kirjs** [comment]
@remy I added URL-check and travis-ci integration, now it's your turn, and then it's ready to get in. 
I pinged you on hangouts, or what would be the best way for a quick chat?

**Comment by remy** [comment]
Sorry, picking this up this morning (been busy with client work). I don't use the @ leftlogic hangouts at all (sorry). I sometimes use gitter – but am rarely on it, but happy to chat there. What timezone are you in? (I'm UK).

**Comment by remy** [comment]
Hey, so I've never had much luck with saurcelabs (mostly because I'd experiment but never had a real end goal in mind). I've given travis the auth details in the environment (under `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`) but the test is failing saying that saucelabs got `none` for the username + key. I'm not sure where this is being read from...

Anyway, if you want to ping me, I'll have https://gitter.im/jsbin/jsbin open all day.

**Pull request merged**